### PR TITLE
Fixes #18338: Rudder SSL default configuration should follow the system default one

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/18338

Replacing previous PR: https://github.com/Normation/rudder/pull/3263

### CODE REVIEW

File: relay/sources/apache/rudder-vhost.conf [hunk 1]
```diff
@@ -34,3 +34,8 @@
 
   IncludeOptional /opt/rudder/etc/rudder-apache-*-ssl.conf
 </VirtualHost>
+
+SSLProtocol             all -SSLv3 -TLSv1 -TLSv1.1
```
Comment (amousset) [inline relay/sources/apache/rudder-vhost.conf:38]: please add a link to the mozilla config page with the settings you used

Comment (amousset) [reply relay/sources/apache/rudder-vhost.conf:38]: and an explanation for the apache and openssl versions

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**